docs/product-assembly_business-processes/src/main/asciidoc/pam/master-docinfo.xml

```diff
@@ -5,6 +5,6 @@
 
 </subtitle>
 <abstract>
-	<para>In this tutorial, you will create and test a mortgage application scenario using {PRODUCT} {PRODUCT_VERSION}. This procedures in this document are based on the included Mortgage_Process sample project. To save time, while ensuring that you understand how to create a working business process, you will create some assets from scratch, but will also utilize some of the pre-defined assests contained in the sample project. To expand on particular areas of interest, each chapter contains a link to generic examples of how to complete simliar tasks and expanded conceptual information.</para>
+	<para>In this tutorial, you will create and test a mortgage application scenario using {PRODUCT} {PRODUCT_VERSION}. This procedures in this document are based on the included Mortgage Process sample project. To save time, while ensuring that you understand how to create a working business process, you will create some assets from scratch, but will also utilize some of the pre-defined assests contained in the sample project. To expand on particular areas of interest, each chapter contains a link to generic examples of how to complete simliar tasks and expanded conceptual information.</para>
 </abstract>
```

Comment:
It's not shown in your PR, but the Prereqs under the Preface, the second prereq needs to be "{PRODUCT} is installed" for consistency. Review all prereqs and make sure all are in the consistent passive format.